plugins/MauticCrmBundle/Integration/DynamicsIntegration.php

```diff
@@ -413,7 +413,7 @@ public function pushLead($lead, $config = [])
      */
     public function getLeads($params = [], $query = null, &$executed = null, $result = [], $object = 'contacts')
     {
-        if ('Contact' === $object) {
+        if ('Contacts' === $object) {
```

Comment:
Need check both Contact and Contacts - see our PR https://github.com/mautic/mautic/pull/5674/files#diff-739ed4b59971b6352ae731f734796c5fR438
Both fixes are in uncomming big update https://github.com/mautic/mautic/pull/5674. @escopecz  consider duplicate label

Reply:
Yep, #5674 seems to consider more use cases. I'll close this one.